app/models/ability.rb

```diff
@@ -88,12 +88,13 @@ def signed_in_with_roles(user)
     # Abilities from not_signed_in and signed_in are also inherited
     signed_in(user)
 
-    signed_in_with_organizer_role(user)
-    signed_in_with_cfp_role(user)
-    signed_in_with_info_desk_role(user)
-    signed_in_with_volunteers_coordinator_role(user)
+    signed_in_with_organizer_role(user) if user.has_role? :organizer, :any
+    signed_in_with_cfp_role(user) if user.has_role? :cfp, :any
+    signed_in_with_info_desk_role(user) if user.has_role? :info_desk, :any
+    signed_in_with_volunteers_coordinator_role(user) if user.has_role? :volunteer_coordinator, :any
 
     # for users with any role
+    can :access, Admin
```

Comment:
That was happening already actually, all roles (organizer, cfp, info desk, volunteers coordinator) have access to the admin area (index Conference, which is inherited by https://github.com/openSUSE/osem/blob/master/app/models/ability.rb#L36). That's all the Admin option does (just a manual thing we thought of so that we can easily check if a user has any of those roles instead of typing a long if statement, https://github.com/raluka/osem/blob/view_comment_notification/app/models/ability.rb#L97)